test/metrics.bats

```diff
@@ -97,7 +97,7 @@ function teardown() {
 	CONTAINER_ENABLE_METRICS=true CONTAINER_METRICS_PORT=$PORT start_crio
 
 	jq '.linux.resources.memory_limit_in_bytes = 15728640
-        | .command = ["sh", "-c", "dd if=/dev/zero of=/dev/null bs=20M"]' \
+        | .command = ["sh", "-c", "sleep 5 && dd if=/dev/zero of=/dev/null bs=20M && sleep 30"]' \
```

Comment:
The second sleep makes no sense since `dd` is running until killed.

The first sleep is added by #7036 (although I'm using `sleep 1` in there. Can we please merge #7036?

The problem with the flake is, sometimes neither conmon nor systemd detect an OOM event for cgroup v2. I am not entirely sure why.

Reply:
Ah I did not see #7036, sorry about that.